<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tsoxic Code Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="te_code_area">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>900</width>
      <height>400</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_compilation_log">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>900</width>
      <height>100</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_status">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>1010</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_new">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>110</width>
      <height>62</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_open">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>62</y>
      <width>110</width>
      <height>62</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_save">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>124</y>
      <width>110</width>
      <height>62</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_copy">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>186</y>
      <width>110</width>
      <height>62</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_paste">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>248</y>
      <width>110</width>
      <height>62</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_cut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>110</width>
      <height>62</height>
     </rect>
    </property>
    <property name="text">
     <string>Cut</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_compile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>372</y>
      <width>110</width>
      <height>62</height>
     </rect>
    </property>
    <property name="text">
     <string>Compile</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_team">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>434</y>
      <width>110</width>
      <height>62</height>
     </rect>
    </property>
    <property name="text">
     <string>Team</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
